cmd/monaco/v2/deploy/deploy.go

```diff
@@ -175,13 +179,13 @@ func printErrorReport(deploymentErrors []error) {
 	groupedConfigErrors := groupConfigErrors(configErrors)
 
 	for project, apiErrors := range groupedConfigErrors {
-		log.Error("==== Project `%s`\n", project)
+		log.Error("==== Project `%s`", project)
```

Comment:
Please undo those changes, as they are completely unrelated to the original bug.